tools/build/watch.js

```
@@ -75,13 +72,15 @@ function watch(globs, opts, tasks) {
   }
 
   function tasksDone(err) {
-    if (err) throw err;
     if (eventsRecorded) {
       // eventsRecorded has increased during the run, run again on the next turn
       timeoutId = setTimeout(invokeCallback, 0);
     } else {
       timeoutId = null;
     }
+    if (err) {
+      watcher.emit('error', err);
```

what is this intended to do?

chokidar doesn't care and nobody else is listening for these errors on chokidar.

wouldn't it be better to rethrow the error at the end?